components/styled/bookshelf.ts

```diff
@@ -173,6 +173,17 @@ text-gray-500
   justify-around
 `
 
+const ReadingHeader = tw.div`
+  mt-5
+  text-xl
+`
+
+const HeaderLink = tw.a`
+text-[#3F51B5]
```

Comment:
Please indent

Reply:
got it. I think the Tailwind Intellisense is whats confusing me because it shows a color preview on the left side and its taking up space so I thought it was indented when it actually wasn't. hehe

like this 
![image](https://user-images.githubusercontent.com/91857581/188838460-2f76b650-bdce-46be-9e5a-4367a79558fc.png)

Reply:
No worries man